### DISCUSSION TITLE
Who wants to be hired? (Mar 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#34948] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #33870. We've had more [*success stories*](https://twitter.com/Nutlope/status/1475929984174895105) of developers being hired!* 

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #34949.

**Comment by abe312** [comment]
**Need help with incremental migration to next.js? or parallel deployment(where new code, old code and some CMS is running on the same domain)?**

Looking for full-time remote roles in a well-funded stable company (I will work in the company's core office hours. 0 issues with timezones. Been working remotely before covid)

Building websites since 2016, working with React since 2018 and have 2 years of production next.js experience

Super comfortable with Next.js, React, Redux (with sagas), tailwind, material UI, styled-components, AMP, SEO schemas, scalable deployment on GCP/AWS/Vercel & MERN

I value web performance (web vitals) and accessibility and could create high conversion dynamic pages

website: https://abhineet.me/
email: contact+vercel@abhineet.me

**Comment by G3root** [comment]
👋 Hey Everyone, 
I'm Nafees Nazik a self taught developer with 2 year's experience looking for a full-time role.

Location: Sri Lanka

Remote: Yes

Willing to relocate: Maybe

Technologies: React, Next.js, React Native, GraphQL, Prisma, Tailwind CSS, CSS-in-JS, Trpc, RestApi, Typescript

Resume/CV: https://github.com/G3root/G3root/blob/main/Resume-new.pdf

Website: https://www.nafeez.xyz/

Email: nafeesnazik21@gmail.com

**Comment by crackwebai1024** [comment]
Remote : Yes
Willing to relocate: No
working timezone : client's timezone (including UK, EST, CST, PST)
Looking for : fully remote job (full-time)
mainskills : 
 - Frontend 
     (SPA - React, Angular, Vue, Svelte)
     (SSR - Next, Nuxt, Sapper)
     (UI - material ui, tailwind CSS, Chakra UI, styled-components, vuetify)
 - Backend (Node - express, Python - django, aws Lambda(chalice, nodejs))
portfolio : http://mingxing.vercel.app

**Comment by andrepith** [comment]
Hello everyone, my name is Andre, a Frontend Developer with 3+ years experience with 1+ year Fullstack experience and also a Bachelor of Computer Science degree

Location: Indonesia
Remote: Yes
Willing to relocate: Yes
Technologies: ReactJS, React Native, Redux, GraphQL, Storybook, Styled-Component, NodeJS, Express, NextJS. MongoDB
Resume/CV: https://techrez.io/resume/andre-adikara
Website: https://www.andreadikara.com/
Email: andrepith@yahoo.co.uk
Github: https://github.com/andrepith

**Comment by kellenwiltshire** [comment]
Hi Everyone,

My name is Kellen and I am a self-taught Web Developer looking for the next step in my journey. I have taught myself with the help of online courses how to create beautiful and functional websites combining the latest frameworks and technologies. I have knowledge of React, Javascript/Typescript, TailwindCSS, Styled Components, as well as traditional HTML5 and CSS3. I am ready for the next step in my Web Development career.

Location: Ontario, Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Emotion, Typescript, Squareup/Shopify
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Comment by AWIXOR-zz** [comment]
Location: Morocco
Remote: Yes
Willing to relocate: Yes
Technologies: Next JS, React, Redux, Recoil, TypeScript/JavaScript, Tailwind ...
Resume/CV: https://awixor.com/resume.pdf
Website: https://awixor.com/
Email: me.awixor@gmail.com

**Comment by andreyscott** [comment]
I'm A Front end JavaScript Dev Looking for full-time/Part-time remote roles in a well-funded stable company .
I love to Code and I have zero issues with time zones. 

Building websites since 2019, working with React since 2019and have 2 years of production next.js experience

Super comfortable with  JavaScript, Next.js, React, Redux (with sagas), tailwind,  Scss,  Styled Components, material UI, styled-components,  Git, GitHub, SEO schemas, Firebase & a little bit of React Native

Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: Next JS, React, Redux, Recoil, TypeScript/JavaScript, Tailwind ...
Resume/CV: https://andrey-profile.web.app/resume
Website: https://andrey-profile.web.app
Email: [me.andreyscott301@gmail.com](mailto:me.andreyscott301@gmail.com)

**Comment by Shivamk99** [comment]
hey @Nutlope 
Hope you are doing well.
I am Shivam, a frontend Developer where I play around with Nextjs, Reactjs with GraphQL, and monorepo. Currently, I am learning typescript as well. Looking forward to new opportunities as Nextjs Developer.

Location: Bangalore, India
Remote: Yes
Willing to relocate: No
Technologies:
 - SPA: Reactjs
 - SSR: Nextjs
 - UI: material UI,  styled-components, 
 - others: GraphQL, Apollo, Typescript
Resume/CV: https://shivam.vercel.app/
Website: https://shivam.vercel.app/
Email: smileyshivam3042@gmail.com

Looking for a positive response. 

Thank you

**Comment by CallmeHongmaybe** [comment]
Hello everyone,
You can call me Dennis Kwan, and I've been focusing on my niche of making blogs from scratch where I either code it with Next.js or build it from the ground in Wordpress.

Location: Ho Chi Minh City, Vietnam
Remote: Yes
Willing to Relocate: Yes if the opportunity is right
Technologies: Next.js, React, Tailwind CSS, Material UI, Sanity CMS, Wordpress, and JAM stack.
Website: [denniskwan.vercel.app](https://denniskwan.vercel.app/)
Resume on LinkedIn: https://www.linkedin.com/in/quan-hong/
email: [quanvihong@gmail.com](mailto:quanvihong@gmail.com)

**Comment by nyambura00** [comment]
Winnie Nyambura on deck👩‍💻🇰🇪

I hope this route leads me to a potential work environment that is progressive, determined, humble and hardworking. I can't wait to grow with you!

Location: Eldoret, Kenya

Remote: Yes

Willing to relocate: Yes

Technologies: React ecosystem - Nextjs, TailwindCSS, Styled-components, Animations, Redux, Graphql, Data loaders, Solidity, Hardhat, Truffle, Polygon, Ethereum.

Resume/CV: https://www.github.com/nyambura00

Email: winnienyambura14@gmail.com

I am proactive, smart and I have a fetish for performing technologies.

**Comment by ariesclark** [comment]
I'm Aries, a 19 year old Canadian citizen with a burning passion for software development and improving the state of the web. I started software development when I was quite young — 13 years old in fact. In those 6 years I've done so many different things ranging from developing Minecraft mods in Java, writing quite a few horrible PHP websites, and all the way to where I am now. As of late, my focus has been on using modern languages like TypeScript to create web based applications and services with React and Next.js among other libraries.

I'm always open for an opportunity. Whether you have a question or just want to say hi, 
I'll try my best to get back to you! Take the chance and message me.

---

Location: Alberta, Canada
Remote: Yes
Willing to relocate: If given the right opportunity, yes.
Technologies: 
1. TypeScript / JavaScript (ES6+)
4. React and related libraries
5. Next.js (obviously ❤️)
6. Tailwind CSS 
7. and much much more.

Website: https://ariesclark.com/
Email: ``me@ariesclark.com``

**Reply by atolz** [reply]
Nice portfolio man🏅💯

**Comment by olaleykhan** [comment]
Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: React, NextJs, Vue, Typescript, ExpressJs
Website: https://lekan.dev
Email: analystabdulfatah@gmail.com

**Comment by codewithPoppy** [comment]
Hello everyone,
I am Yevhenii Molchanov, and I am an experienced and meticulous Front-end developer with overall 8+ years of experience and passion for responsive website design writing top-quality code. Fully attracted to React and have been developing many React/Next.js websites.

Location: Sydney, Australia
Remote: Yes
Willing to Relocate: Maybe
Technologies: Next.js, React, Tailwind CSS, Material UI, Contentful CMS, Blitz.js, Node.js, Angular, Typescript
Resume: https://drive.google.com/file/d/1hnJ41tPsKtIkdDbc6L-ei4CU8BaZRk8l/view?usp=sharing
email: challenger0920@outlook.com

Thanks.

**Comment by LawrenceMueller** [comment]
Hello, my name is Lawrence Mueller. I am a web developer and have been developing for three years. I am at a junior level and would love to get started somewhere.

Location: Pomona, California, United States
Remote: Yes
Willing to relocate: Maybe
Technologies: Next, React, Node, MongoDB, Redis Enterprise, GitHub, CSS, HTML 
Resume/CV: https://drive.google.com/file/d/1eVLVsWlhExanmYrdxJylTWbT0TGegaFJ/view
Website: https://www.lawrencemueller.com/
Email: lawrencemueller18@gmail.com

**Comment by shimaxu** [comment]
Hi,

My name is Mohamed Shimaz and self thought developer. I have 6 plus experience in the field of software engineering developing enterprise applications in government agencies. Moreover, I have experienced in developing various client project in teams.

Location: Maldives - GMT+5
Remote: Yes
Willing to relocate: Maybe
Technologies: Next JS / React / JavaScript / TypeScript / GraphQL / Nexus / Node / Php (Laravel), CSS(Tailwind & Ant Design)
Resume/CV: https://drive.google.com/file/d/1CyowD2pIlYy54hSkI_JMSpNr7GB1efeo/view
Email: shimax15@gmail.com

**Comment by adnanlah** [comment]
Hello,

My name is Adnan Lahrech and I have been a freelance web developer for few years. I have worked with Vue, React, Next, Gatsby, Node, Express, Socket.IO, Web crawling, Electron, databases and ORMs, WebGL (2D), SVGs (Inkscape), and even PHP and Laravel back in the day. I also have a masters degree in CS: Image processing (graduated in 2016).

I am looking for any gig now as a remote (preferably a React related one) and I can work with any time-zone, so I would be happy if you check my Github, and my Codepen: https://codepen.io/dinopens.

I can understand both English and French. Reach me out on Discord: DINO#9827 or via E-Mail: lahadn@gmail.com.

Thanks.

**Comment by AndrewRoy3535** [comment]
Hi everyone,
This is Andrew Roy, lives in Dhaka, BD. Currently working as a freelance developer. I have 3 year+ working experience. React, Next.js, vue.js, Node.js, React-native, Flutter, Sanity are some of technologies that am working with right now. I am also very flexible to switch to any new technology or language.

Location: Dhaka, Bangladesh
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next, Sanity, Tailwind, Vue, etc...
Resume/CV: https://drive.google.com/file/d/1momHOdFebKcJVTGXi5tCWwGyQmlZGFL2/view?usp=sharing
Website: https://andrew-roy.vercel.app
Email: eda.twinkle@gmail.com

**Comment by fulltime117** [comment]
Hello.
Hope you are doing great.

Location: China
Remote: Yes
Willing to relocate: No
Technologies: Next, React,  Angular,  Node, Styled Component, Tailwind CSS, SCSS, 
Résumé/CV: https://drive.google.com/file/d/1iWzvSCXWzj5aXPk0d4G2lTzRUSO1mEWc/view?usp=sharing
Website: http://nicolae.66ghz.com
Email: nicolaepop055@gmail.com

Thank you.

**Comment by realtouseef** [comment]
👋Hi there, I am Touseef, my role is to write code that meets the requirements of your mocks and fulfills your user stories. I have a knack for creating minimal websites and have an eye for even the smallest of details. I’m a big fan of simplicity and am a great team leader as compared to an individual contributor.

🏡Location: Pakistan
🗺Remote: Yes
✈Willing to relocate: Yes, if sponsored
👨‍💻Technologies: SCSS, TailwindCSS, JavaScript, ReactJS, NextJS, GraphQL
📑Resume/CV: https://docs.google.com/document/d/1UocXZEGoIzAh1_8K6TVIWq48YmZig6__Ov6UHgcLRpg/edit?usp=sharing
🟢Website: https://touseef.vercel.app
📧Email: touseef69r@gmail.com

**Comment by Ibrahim-Isah** [comment]
> # What is this?
> This discussion is for Next.js developers looking for work.
> 
> # Candidates
> Share your information if you are looking for work. Please use this format:
> 
> Location: Remote: Willing to relocate: Technologies: Resume/CV: Website: Email:
> 
> Good luck with interviewing!
> 
> _If you're looking for last month's discussion, see #33870. We've had more [_success stories_](https://twitter.com/Nutlope/status/1475929984174895105) of developers being hired!_
> 
> If you got hired through this job board, please reach out to me ([hassan@vercel.com](mailto:hassan@vercel.com)) so I can send you some swag :)
> 
> If you're a company looking to post a role, use #34949.

Hey Everyone,
I'm Isah Abba Ibrahim a self taught developer with 2 year's experience looking for a full-time role.

Location: Nigeria

Remote: Yes

Willing to relocate: Maybe

Technologies: React, Next.js, React Native, Tailwind CSS, CSS-in-JS, RestApi, Typescript, Openapi.

Resume/CV: https://drive.google.com/file/d/1Y08NjLScSbfed7bnSXOaQ3tu3Gn5J2Zl/view?usp=drivesdk

Website: https://portfolio-one-three.vercel.app/

Email: isahibn08@gmail.com

**Comment by ESMAT001** [comment]
Hello World👋
I am Esmatullah Niazi a.k.a Skywalker😉 a Sofware Engineer with an incurable curiosity. And an enthusiastic and hard-working developer with 3 years of experience that does his job passionately. As I say the secret to great achievement is not talent, but a passionate persistence.

Location: Afghanistan (heart 💖 of Asia)
Remote: Yes 100%
Willing to relocate: Maybe
Technologies: Tailwind CSS, Bootstrap, Reactjs, NextJs, ExpressJs, MongoDB, Python, Scrapy AND .. .  .  .   .
Resume/CV: https://skywalker.web.app/EsmatullahNiaziCV.pdf
Website: https://skywalker.web.app/
Email: [esmat10.niazi@gmail.com](mailto:esmat10.niazi@gmail.com)

![portfolio](https://user-images.githubusercontent.com/56363653/158861271-be27030c-b695-4200-8503-06bef5f306c3.PNG)

**Comment by simonhlee97** [comment]
Location: from US, currently living in South Korea
Remote: Yes (preferred)
Willing to relocate: yes
Technologies: JavaScript, NodeJS, ReactJS, VueJS, Next.js, MongoDB, Postgres, TailwindCSS, Express
Resume/CV: [resume](https://github.com/simonhlee97/career-docs/blob/main/resume.simon.lee.pdf)
Website: [home page](https://simonhlee97.github.io/)
Email: simonhlee4945@gmail.com

**Comment by euaaron** [comment]
Location: Juiz de Fora/Brazil

Remote: preferable

Willing to relocate: Are you willing to pay and help with the relocation (including the visa if needed)? If so, let's do it!

Technologies: Programming logic applied to every technology that has a readable documentation or a code example. I'm a generalist developer. I like React, Typescript and C#, but I'm not tied to any role or technology.

Resume/CV: Take a look at https://linkedin.com/in/aaron-carneiro/

Website: https://aaroncarneiro.com

Email: hire@aaroncarneiro.com

**Comment by tawaliou** [comment]
Hi my name is Tawaliou ALAO a web developer. I'm open for web dev role or internship in web development

Location: Benin, Africa
Remote: Yes
Willing to relocate: Yes
Technologies: React.js, Next.js, TypeScript, styled-component, Chakra UI, Node.js  
Resume/CV: https://drive.google.com/file/d/1VI7D0IcoTXaKK22f4pes6M24qbI0OErP/view?usp=sharing
Website: https://tawaldevuniverse.vercel.app/
Email: alaotawaliou@gmail.com

**Comment by pnoken** [comment]
Location: Accra
Remote: Yes
Willing to relocate: Yes
Technologies: React.js, Next.js, TypeScript, styled-component, Ant-D, Framer motion, TailwindCSS
Resume/CV: [LinkedIn](https://www.linkedin.com/in/pnokn/)
Website: https://pnortey.com
Email: norteypeter@gmail.com

**Comment by chemiadel** [comment]
Location: Any
Remote: Yes
Willing to relocate: No
Technologies: TS, Nextjs, react-native, RESTful, Graphql
Resume/CV: https://drive.google.com/file/d/1QolWteIku6cZSzjEuQlvUh8qeAFkwIHE/view?usp=sharing
Email: adelchemi6@gmail.com

**Comment by biztex** [comment]
Hello, dear.
I am Antonio from Italy.
And I am a full stack developer.

Location: Italy
Remote: Yes
Willing to relocate: Yes
Technologies: Next, React, Node.
Resume/CV: https://drive.google.com/file/d/1_zhDKeJqn7iKhfVxQLe1vtco1zlPFySn/view?usp=sharing
Email: luxuryman887@gmail.com

Thank you.

**Comment by bioold** [comment]
Hi! I'm working as a lead developer for several projects now!
I've been a programmer for 10+ years, currently specializing in Web3, Solidity & Rust after enough experience in React.js & Node.js. 
Furthermore, I have a passion for learning new skills, and I'm extremely dedicated to my work!  Team says, "We can't do anything without Bio."
Looking for a new position with full of challenges with a friendly team.

**Location:** US/Japan
**Remote:** Remote
**Willing to relocate:** No
**Technologies:** 

- Blockchain - 3+ years
- Node - 8+ years
- React - 6+ years
- Next - 3+ years
- Other relevant skills

**Resume/CV:** [Here](https://drive.google.com/file/d/1Qctsf_Ik5PHO_TsLm_IyhcHwtakThSVQ/view?usp=sharing)
**Email:** bio.star1001@gmail.com

**Comment by gyenabubakar** [comment]
I work at Walulel where I build geocivic platforms with the Vue and Nuxt frameworks, but I also have experience working with TypeScript, React and Next.js, and I am currently learning Go and MongoDB. I have 3 years of experience, an eye for building intuitive and engaging user interfaces, the passion to learn and am always appreciative of constructive criticism.

**Location:** Worldwide
**Remote:** Yes
**Willing to relocate:** Yes (if Visa would be sponsored)
**Technologies:**

- JavaScript - 3+ years
- React - 3+ years
- Next.js - 1 year
- Vue - 2 years
- Nuxt - 2 years

**Resume/CV:** [Here](https://drive.google.com/file/d/1EkDd0otYlV6MgIrsCiqqCu3I26O6vgRx/view?usp=sharing)
**Website:** https://gyen.dev
**Email:** gyenabubakar@gmail.com

**Comment by rajibola** [comment]
Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: 
  * JavaScript - 4+ years
     React/NextJS/GatsbyJS - 4+ years
     React Native- 3+ year
     Typescript - 2+ years
     GSAP - 1+ years
     TailwindCSS - 1+ years

Resume/CV:  [view my CV](https://drive.google.com/file/d/1w6IgpEVrZTYEkEZFLI5gf8c3qbAV-sMW/view?usp=sharing)
Website: [Still in development](https://aridwan.netlify.com)
Email:  [rajibola1997@gmail.com](rajibola1997@gmail.com)

**Comment by kumard3** [comment]
Location: India
Remote: Yes (preferred)
Willing to relocate: yes
Technologies: JavaScript, NodeJS, ReactJS, VueJS, Next.js, MongoDB, Postgres, TailwindCSS, Express
Resume/CV: [resume](https://drive.google.com/file/d/1XyonW6W6muIN7tcAUUDhRny6tFGPEgtv/view?usp=sharing)
Website: https://kumard3.me
Email: [kumardeepanshu157@gmail.com](mailto:kumardeepanshu157@gmail.com)

**Comment by albertkimdev** [comment]
Location: Thailand
Remote: Yes
Willing to relocate: No
Technologies: React.js/Next.js/Express/Nodejs and many other tools in this ecosystem.
Resume/CV: https://www.albertkim.co/
Website: https://www.albertkim.co/
Email: albertkim91@gmail.com

Looking for contract/freelance work.

I built a website that makes websites. I'm a full-stack React.js and Nodejs developer with advanced frontend skills. If you need someone who can contribute to your React.js project, lets talk!

Some work of mine:
https://www.albertkim.co/
https://leavewithaweb.site/
https://verifiedink.us/create

**Comment by magiobus** [comment]
Magio Here ✌️ 
I'm Searching for a job with a NextJS Stack 🤓  

Location: Mexico
Remote: Yes
Willing to relocate: Maybe, if a work visa is sponsored.
Technologies: Javascript, React, NodeJs, Nextjs, TailwindCSS, MongoDB, Firebase
Resume/CV: https://magiobus.com/cv.pdf
Website: https://magiobus.com
Email: magio@magiobus.com

I made an on demand merch website for latinamerica content creators using NextJS + mongodb + stripe + cloudinary
https://ninogamer.com

**Comment by kchikech** [comment]
Hello ! 

I am hamza, I am Mid FullStack developer. 

Location: Casablanca, Morocco
Remote: Yes
Willing to relocate: Yes
Technologies: ReactJs, ReactNative, NextJs, Redux, PostgreSQL, MongoDb, Prisma, Prisma, NodeJs, ExpressJs, Jest ... 
Resume/CV: https://kchikech.me
Website: https://kchikech.me
Email: hamza@kchikech.me

**Comment by eric-burel** [comment]
Hi fellow recruiters and entrepreneurs :vulcan_salute: ,

I am a seasoned JavaScript engineer with experiences in building startup products with Next.js, Electron, GraphQL, Meteor...
especially in the field of Big Data visualization and exploration.

My latest work is moving the Vulcan open-source project to a modern "Monorepository".

This architecture is excellent if your company must manage multiple JavaScript applications that may share some common code.

:man_technologist:  Demo here: https://github.com/VulcanJS/vulcan-npm

I can help you moving your company to a similar Monorepo architecture (among other things :)).

My hourly rate: 120$. I am located in France and can work in French or English speaking environments.

You can reach me out via mail: eb@lbke.fr

Looking forward to working with you!

Location: France
Remote: Yes
Willing to relocate: No (I am a freelance)
Technologies: Next.js, Remix, GraphQL, React, Python, Electron
Resume/CV: My publications: https://medium.com/@eric.burel ; my GitHub with OSS projects:  https://github.com/eric-burel and my previous freelance experiences (in French :fr:) https://www.lbke.fr/eric-burel
Website: https://www.lbke.fr
Email: eb@lbke.fr

**Comment by jayantkageri** [comment]
Location: Sirsi, India
Remote: Yes
Willing to relocate: No
Technologies: NextJS, ReactJS, TailwindCSS, Bootstrap, Python, MongoDB
Resume/CV: [LinkedIn](https://linkedin.com/in/jayantkageri)
Website: https://jayantkageri.in/
Email: jayantkageri@gmail.com

**Comment by coltonbonner** [comment]
Hello! I'm Colton, entry-level full-stack dev
Aspiring to be in the full-stack arena

Location: California, US
Remote: Yes
Willing to relocate: Yes
Technologies: ReactJS, NextJs, TailwindCSS, NodeJS, ExpressJS, PostgreSQL
Resume/CV: [Technical Resume](https://docs.google.com/document/d/1PxfPI_TDjr7asqaQIwj8-0aQ8MEnBH-ee3d2SKLYsrg/edit?usp=sharing)
Website: https://bwre.tech/
Email: colton@bwre.tech
